src/ui/components/shared/LoomModal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { connect, ConnectedProps } from "react-redux";
+import * as actions from "ui/actions/app";
+import { selectors } from "ui/reducers";
+import { UIState } from "ui/state";
+import Modal from "./NewModal";
+
+function LoomModal({ hideModal, modalOptions }: PropsFromRedux) {
+  if (!modalOptions || !modalOptions?.loom) {
+    return null;
+  }
+  return (
+    <Modal options={{ maskTransparency: "translucent" }} onMaskClick={hideModal}>
+      <div className="rounded-lg overflow-hidden" style={{ width: "940px", height: "680px" }}>
+        <div style={{ position: "relative", paddingBottom: "62.93706293706294%", height: 0 }}>
```

Comment:
Where's the paddingBottom and height coming from?

Reply:
looms embeds